tests/unit/dataactbroker/test_sam_recipient.py

```diff
@@ -158,67 +158,67 @@ def test_load_duns(database):
+            'last_sam_mod_date': str(recipient_obj.last_sam_mod_date) if recipient_obj.last_sam_mod_date else None,
+            'deactivation_date': str(recipient_obj.deactivation_date) if recipient_obj.deactivation_date else None,
+            'legal_business_name': recipient_obj.legal_business_name,
+            'address_line_1': recipient_obj.address_line_1,
+            'address_line_2': recipient_obj.address_line_2,
+            'city': recipient_obj.city,
+            'state': recipient_obj.state,
+            'zip': recipient_obj.zip,
+            'zip4': recipient_obj.zip4,
+            'country_code': recipient_obj.country_code,
+            'congressional_district': recipient_obj.congressional_district,
+            'business_types_codes': recipient_obj.business_types_codes,
+            'business_types': recipient_obj.business_types,
+            'dba_name': recipient_obj.dba_name,
+            'ultimate_parent_uei': recipient_obj.ultimate_parent_uei,
+            'ultimate_parent_unique_ide': recipient_obj.ultimate_parent_unique_ide,
+            'ultimate_parent_legal_enti': recipient_obj.ultimate_parent_legal_enti,
+            'historic': recipient_obj.historic
         }
     assert duns_results == expected_duns_results
 
     uei_results = {}
     # Get UEI results
-    for duns_obj in sess.query(DUNS).filter(DUNS.uei.isnot(None)).all():
-        uei_results[duns_obj.uei] = {
-            'uei': duns_obj.uei,
-            'awardee_or_recipient_uniqu': duns_obj.awardee_or_recipient_uniqu,
-            'registration_date': str(duns_obj.registration_date) if duns_obj.registration_date else None,
-            'activation_date': str(duns_obj.activation_date) if duns_obj.activation_date else None,
-            'expiration_date': str(duns_obj.expiration_date) if duns_obj.expiration_date else None,
-            'last_sam_mod_date': str(duns_obj.last_sam_mod_date) if duns_obj.last_sam_mod_date else None,
-            'deactivation_date': str(duns_obj.deactivation_date) if duns_obj.deactivation_date else None,
-            'legal_business_name': duns_obj.legal_business_name,
-            'address_line_1': duns_obj.address_line_1,
-            'address_line_2': duns_obj.address_line_2,
-            'city': duns_obj.city,
-            'state': duns_obj.state,
-            'zip': duns_obj.zip,
-            'zip4': duns_obj.zip4,
-            'country_code': duns_obj.country_code,
-            'congressional_district': duns_obj.congressional_district,
-            'business_types_codes': duns_obj.business_types_codes,
-            'business_types': duns_obj.business_types,
-            'dba_name': duns_obj.dba_name,
-            'ultimate_parent_uei': duns_obj.ultimate_parent_uei,
-            'ultimate_parent_unique_ide': duns_obj.ultimate_parent_unique_ide,
-            'ultimate_parent_legal_enti': duns_obj.ultimate_parent_legal_enti,
-            'historic': duns_obj.historic
```

Comment:
Including here, could line 250 be updated to `# not included in any of the exec comp but listed in sam_recipient`?